### PR TITLE
Ugly_Numbers

### LINKED ISSUE #1036: Nth Ugly Number in ruby
Sir i would like to implement nth ugly numbers in ruby language.

### PR DESCRIPTION
**Pull request by AnshitaVishwa**
Fixes #1036 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Added the Nth Ugly Number in Ruby.

#### Languages Used:
- Ruby

#### Files Added:
- Nth_Ugly_Number.rb

### CODE REVIEW

**Comment by AnshitaVishwa** [comment]
Kindly review it sir.

**Comment by marwahmanbir** [comment]
@AnshitaVishwa mention the issue number in the PR template above (Fixes #?)

**Comment by AnshitaVishwa** [comment]
@marwahmanbir done sir

**Comment by somya-kapoor** [comment]
@AnshitaVishwa Please add issue number in the PR template.

**Comment by AnshitaVishwa** [comment]
I have added the issue number.

**Comment by somya-kapoor** [comment]
@AnshitaVishwa Added issue number in PR description (template ).

**Comment by AnshitaVishwa** [comment]
@somya-kapoor  I am really sorry. I have done it right now .

File: Ugly_Number/Nth_Ugly_Number.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          elsif x % 3 == 0
+            x = x/3
+          elsif x % 5 == 0
+            x = x/5
+          else
+            m = 0
+            break
+          end
+      end
+      return m
```
Comment (KanchanThareja) [inline Ugly_Number/Nth_Ugly_Number.rb]: convert to 4 space indentation

File: Ugly_Number/Nth_Ugly_Number.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+n = gets.chomp.to_i
+i = 1
+
+# Finds the Nth ugly Number
+while n != 1
+    if Is_Ugly(i+1) == 1
+      n = n-1
+    end
+    i = i+1
+end
```
Comment (KanchanThareja) [inline Ugly_Number/Nth_Ugly_Number.rb]: Here as well

**Comment by AnshitaVishwa** [comment]
Ma'am I have done the changes.

**Comment by jainaman224** [comment]
@KanchanThareja Please review.

**Pull request merged**